lua/nvim-treesitter-playground/query_linter.lua

```diff
@@ -54,12 +58,14 @@ function M.lint(buf)
   local filename = api.nvim_buf_get_name(buf)
   local ok, query_lang = pcall(vim.fn.fnamemodify, filename, ":p:h:t")
   query_lang = filename ~= "" and query_lang
-  local query_lang = ok and query_lang
+  query_lang = ok and query_lang
   if not query_lang then
     query_lang = query_lang_from_playground_buf(buf)
   end
+
   local ok, parser_info = pcall(vim.treesitter.inspect_language, query_lang)
-  local parser_info = ok and parser_info
+
+  parser_info = ok and parser_info
 
   local matches = queries.get_matches(buf, "query-linter-queries")
 
```

Comment:
Maybe it would be be good to rename `buf` to `query_buf`. This would make it more difficult to forget that buf is actually a query buffer and make the function signature more readable.

Reply:
Really awesome work this PR!